src/resizer/resizer.ts

```diff
@@ -114,6 +114,10 @@ export default class Resizer<C extends IConfig = IConfig> {
     }
 
     private onMouseDown = (event: MouseEvent): void => {
+        const LEFT_MOUSE_BUTTON = 0;
+        if (event.button !== LEFT_MOUSE_BUTTON) {
```

Comment:
Optional bonus points: Rename to `PRIMARY_MOUSE_BUTTON` for left-handed layouts, touchpads and other input devices.